cmd/xl-v1-utils.go

```diff
@@ -301,6 +310,8 @@ func readXLMeta(ctx context.Context, disk StorageAPI, bucket string, object stri
 	// obtain xlMetaV1{} using `github.com/tidwall/gjson`.
 	xlMeta, err = xlMetaV1UnmarshalJSON(ctx, xlMetaBuf)
 	if err != nil {
+		logger.GetReqInfo(ctx).AppendTags("disk", disk.String())
```

Comment:
I can also add the bucket and object information here

Reply:
Yes. but bucket/object info should be added at the topmost caller.

Reply:
The topmost caller will definitely not have object, may have bucket information. So, in this scenario, passing down the topmost context will not be helpful. I am trying to see if I can add relevant information in between, the topmost caller and at this level.